install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+<2> The image run by this pod.
+<3> The name of the volume. This name must be the same in both the `containers` and `volumes` sections.
+<4> The mount path as seen in the container.
+<5> The group id to be assigned to the container.
+<6> The path to the storage on the NFS server
+<7> The NFS server
+====
+
+Save the pod definition to a file, for example *_nfs-2.yaml_*,
+ and create the pod:
+
+====
+----
+# oc create -f nfs-2.yaml
+pod "busybox-nfs-pod" created
+
+#verify pod was created
+# oc get pods
+NAME                READY     STATUS    RESTARTS   AGE
+busybox-nfs-pod     1/1       Running   0          3s
+----
+====
+
+More details are shown in the `oc describe pod` command:
+
+====
+----
+[root@ose70 nfs]# oc describe pod busybox-nfs-pod
+Name:				busybox-nfs-pod
+Namespace:			default
+Image(s):			busybox
+Node:				ose70.rh7/192.168.234.148
+Start Time:			Mon, 21 Mar 2016 10:19:46 -0400
+Labels:				name=busybox-nfs-pod
+Status:				Running
+Reason:				
+Message:			
+IP:				10.1.0.5
+Replication Controllers:	<none>
+Containers:
+  busybox-nfs-pod:
+    Container ID:	docker://346d432e5a4824ebf5a47fceb4247e0568ecc64eadcc160e9bab481aecfb0594
+    Image:		busybox
+    Image ID:		docker://17583c7dd0dae6244203b8029733bdb7d17fccbb2b5d93e2b24cf48b8bfd06e2
+    QoS Tier:
+      cpu:		BestEffort
+      memory:		BestEffort
+    State:		Running
+      Started:		Mon, 21 Mar 2016 10:19:48 -0400
+    Ready:		True
+    Restart Count:	0
+    Environment Variables:
+Conditions:
+  Type		Status
+  Ready 	True 
+Volumes:
+  nfsvol:
+    Type:	NFS (an NFS mount that lasts the lifetime of a pod)
+    Server:	nfs.f22
+    Path:	/opt/nfs
+    ReadOnly:	false
+  default-token-32d2z:
+    Type:	Secret (a secret that should populate this volume)
+    SecretName:	default-token-32d2z
+Events:
+  FirstSeen	LastSeen	Count	From			SubobjectPath				Reason		Message
+  ─────────	────────	─────	────			─────────────				──────		───────
+  4m		4m		1	{scheduler }							Scheduled	Successfully assigned busybox-nfs-pod to ose70.rh7
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Pulled		Container image "openshift3/ose-pod:v3.1.0.4" already present on machine
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Created		Created with docker id 249b7d7519b1
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Started		Started with docker id 249b7d7519b1
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Pulled		Container image "busybox" already present on machine
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Created		Created with docker id 346d432e5a48
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Started		Started with docker id 346d432e5a48
+----
+[NOTE] 
+====
+Instead of a Persistent Volume Claim being listed under the 'Volumes' section we have 'nfsvol' indicating we used the NFS plugin to access the storage asset.
+====
+
+====
+
+== [Security Requirement] NFS Volume Access
```

Comment:
I think we could probably just drop this section? It's stated in the Overview the example uses NFS, and the NFS doc already has its own NFS-specific "Volume Security" section (https://docs.openshift.com/enterprise/latest/install_config/persistent_storage/persistent_storage_nfs.html#nfs-volume-security), so that seems like enough and that it would just be a repeat here. (Plus there's the full, dedicated "Volume Security" topic at https://docs.openshift.com/enterprise/latest/install_config/persistent_storage/pod_security_context.html, as already mentioned.)